### DISCUSSION TITLE
Roms naming

### DISCUSSION THREAD

**Discussion by Raiper34** [rommapp/romm#669] [Q&A] [answered]
I am just curious if it is possible to scrape information about games, that are not exactly named by IGDB because I have a very large library of games 10K+. I would like to scrape information automatically, but the current scraper finds nothing about my games. My games are named for example `moto-racer-world-tour-u-slus-01321-.zip`. My Batocera Linux scrapes it quite well, but I would like to have also self hosted rom manager.

**Comment by gantoine** [comment]
RomM currently only supports OPL with PS2 games, not PS1, so the above won't match anything. We'll add support in a future release.

**Comment by gantoine** [comment, marked as answer]
Can confirm we'll have this included in 3.1
<img width="332" alt="Screenshot 2024-02-21 at 7 15 09 PM" src="https://github.com/zurdi15/romm/assets/3247106/c6fb463a-2e89-4404-a437-78df826a906a">
